test/EndToEndTests/Tests/Client/Build.Desktop/Microsoft.Test.OData.Tests.Client.csproj

```diff
@@ -222,6 +222,10 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
+  </ItemGroup>

```

Comment:
Why this change?

Reply:
Hmm. I don't know what this is. Must have been added by the IDE while I was changing things. Let me see what happens if I remove it.

Reply:
Removed.